docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc

```diff
@@ -259,6 +259,70 @@ the following configuration should be added as an extra input in the DaemonSet m
 Refer to <<kubernetes-provider,dynamic inputs and Kubernetes provider>> for more information
 about shaping dynamic inputs for autodiscovery.
 
+[discrete]
+== Autodiscover targeted Pods using hints
+
+In order to achieve dynamic monitoring of Pods/containers, one can use hints. This can be achieved by properly
+enabling hints for the `kubernetes` dynamic provider. For more information about this feature refer to
+<<kubernetes-provider,dynamic inputs and Kubernetes provider>>.
+
+In the standalone policy ensure that the feature is enabled:
+
+In order to enable hints one need to add the `hints.enabled: true` in the provider's configuration, ie:
+
+[source,yaml]
+----
+providers:
+  kubernetes:
+    node: "kind-control-plane"
+    hints.enabled: true
+----
+
+Then ensure that the proper volumes and volumeMounts are added properly. These sections are already defined in the
+provided kubernetes manifest, so it's only required to uncomment these sections.
+
+In order to enable hints one need to add the `hints.enabled: true` in the provider's configuration, ie:
+
+[source,yaml]
+----
+providers:
+  kubernetes:
+    node: "kind-control-plane"
+    hints.enabled: true
+----
+
+In order to enable hints one need to add the `hints.enabled: true` in the provider's configuration, ie:
```

Comment:
this section repeats line 271 - https://github.com/elastic/observability-docs/pull/2182/files#diff-b8738d619ef877c7cf3f0dce25b4c512396007f53eddc2d16fe82bfb1a4a1642R271
here should be sentence about uncommenting `volumes and volumeMounts` https://github.com/elastic/observability-docs/pull/2182/files#diff-b8738d619ef877c7cf3f0dce25b4c512396007f53eddc2d16fe82bfb1a4a1642R281-R282, no?